src/main/groovy/org/mockito/release/internal/gradle/RunTestReleaseTask.java

```diff
@@ -0,0 +1,53 @@
+package org.mockito.release.internal.gradle;
+
+import org.gradle.api.DefaultTask;
+import org.gradle.api.logging.Logger;
+import org.gradle.api.logging.Logging;
+import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.OutputFile;
+import org.gradle.api.tasks.TaskAction;
+import org.mockito.release.exec.ProcessRunner;
+
+import java.io.File;
+import java.util.List;
+
+/**
+ * This task run external process and additionally store output of external process to file.
+ */
+public class RunTestReleaseTask extends DefaultTask {
+
+    private static final Logger LOG = Logging.getLogger(RunTestReleaseTask.class);
+
+    private List<String> command;
+    private File buildOutput;
+    private File workDir;
+    private String repoName;
+
+    @TaskAction
+    public void runTest() {
+        LOG.lifecycle("  Run test of {}. The output will be save in {}", repoName, buildOutput.getAbsoluteFile());
```

Comment:
I really like this message! It provides nice context and interesting values that the user/us might want to see.